### PR TITLE
2443: Hide i18next debug logs for test

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

Hide i18next debug logs for test.

### Proposed changes

Hide i18next debug logs for test by overwriting console.log before tests and restoring it afterwards.

### Side effects

None.

### Resolved issues

Fixes: #2443

---

### CODE REVIEW

**Review by LeandraH — APPROVED** [review]
Nice! Still getting a debug statement from `sentry.ts` but I don't think that's in the scope of this ticket, plus it's single lines as opposed to translation objects...

File: web/src/components/__tests__/I18nProvider.spec.tsx [hunk 1]
```diff
@@ -9,9 +9,22 @@ import I18nProvider from '../I18nProvider'
 
 jest.mock('../../utils/BrowserLanguageDetector')
 jest.mock('translations/src/loadTranslations')
+jest.mock('')
```
Comment (f1sh1918) [inline web/src/components/__tests__/I18nProvider.spec.tsx:12]: why we need this?

Comment (steffenkleinle) [reply web/src/components/__tests__/I18nProvider.spec.tsx:12]: we don't, forgot to remove

**Review by f1sh1918 — APPROVED** [review]
looks fine :)

**Pull request merged**